ecosystem/lava-sdk/src/lavasession/consumerSessionManager.ts

```diff
@@ -124,7 +125,7 @@ export class ConsumerSessionManager {
       // For LAVA's initialization, we need to allow the pairing to be updated twice
       // This condition permits the pairing to be overwritten just once for the same epoch
       // After this one-time allowance, any attempt to overwrite will result in an error
-      if (epoch != 0) {
+      if (epoch != 0 && virtualEpoch == 0) {
```

Comment:
cant see anything related to virtual epoch in consumerSessionManager I am assuming not yet finished this flow

Reply:
this is to update the pairing if epoch has not changed, but the virtual != 0(so as not to throw an error)